day_two_guide/environment_backup.adoc

```diff
@@ -0,0 +1,43 @@
+[[day_two_environment_backup]]
+= Creating an environment-wide backup
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+Creating an environment-wide backup involves copying important data
+in case of crashing instances, or corrupt data. Once backups have been created,
```

Comment:
s/Once/After (Style says: once adv Use only to mean “one time.” Do not use as a conjunction to mean “after” or “when.”)